ftplugin/vimwiki/taskwiki.vim

```diff
@@ -45,16 +49,12 @@ augroup taskwiki
 
     " Refresh on load (if possible, after loadview to preserve folds)
     if has('patch-8.1.1113') || has('nvim-0.4.0')
-      execute "autocmd BufWinEnter <buffer> ++once :" . g:taskwiki_py . "WholeBuffer.update_from_tw()"
+      autocmd BufWinEnter <buffer> ++once TaskWikiBufferLoad
     else
-      execute g:taskwiki_py . "WholeBuffer.update_from_tw()"
+      TaskWikiBufferLoad
     endif
 augroup END
 
-" Global update commands
-execute "command! -buffer -nargs=* TaskWikiBufferSave :"      . g:taskwiki_py . "WholeBuffer.update_to_tw()"
-execute "command! -buffer -nargs=* TaskWikiBufferLoad :"      . g:taskwiki_py . "WholeBuffer.update_from_tw()"
```

Comment:
I imagine this could be still used by some users, for example if they want to explicitly update the buffer from TW data, without closing the window?

Reply:
I think we should still keep these two commands, at least for the sake of backwards compatibility.

Reply:
I'm not removing them: https://github.com/tools-life/taskwiki/pull/293/commits/2706bfa0b3dd1bdb45b5846133eab85aa8483d97